matchingengine/OrderBook_test.go

```diff
@@ -96,40 +485,106 @@ func TestLoadOldAskStopLoss(t *testing.T) {
 		stockId:     stockID,
 		askChan:     make(chan *models.Ask),
 		bidChan:     make(chan *models.Bid),
-		asks:        mockAskQueue,
-		bids:        mockBidQueue,
-		askStoploss: mockAskStoplossQueue,
-		bidStoploss: mockBidStoplossQueue,
-		depth:       mockDepth,
+		asks:        testAskQueue,
+		bids:        testBidQueue,
+		askStoploss: testAskStoplossQueue,
+		bidStoploss: testBidStoplossQueue,
+		depth:       testDepth,
 	}
 
-	stopLossAsk := makeAsk(1, stockID, models.StopLoss, stockQuantity, stockPrice, "")
+	ob.LoadOldAsk(ask)
+	ob.LoadOldBid(bid)
+	models.LoadStocks()
+	time.Sleep(time.Second * 2)
```

Comment:
Use wg.SyncGroup


```go
	var wg sync.WaitGroup
	wg.Add(2)

	go func() {
		defer wg.Done()
		ob.LoadOldAsk(ask)
	}()

	go func() {
		defer wg.Done()
		ob.LoadOldBid(bid)
	}()

	wg.Wait()

        models.LoadStocks()
```

Reply:
time.Sleep was not necessary my bad. so just removing it works.